src/DynamoCoreWpf/Utilities/WebView2Utilities.cs

```diff
@@ -36,6 +37,18 @@ public DynamoWebView2() : base()
         /// <returns></returns>
         internal async Task Initialize(Action<string> logFn = null)
         {
+            if (DynamoModel.IsTestMode && !string.IsNullOrEmpty(CreationProperties?.UserDataFolder))
+            {
+                var newPath = Path.Combine(Directory.GetParent(Assembly.GetExecutingAssembly().Location).FullName, "Webview2Data", $"{Environment.ProcessId}", tag);
+                if (Directory.Exists(newPath))
```

Comment:
You mean if directory does not exist?

Reply:
I removed this part. Will try it out in another PR